custom_components/zwave_mqtt/entity.py

```diff
@@ -149,24 +152,37 @@ def device_info(self):
         """Return device information for the device registry."""
         return {
             "identifiers": {(DOMAIN, self.values.primary.node.node_id)},
-            "name": f"{self.values.primary.node.node_manufacturer_name} {self.values.primary.node.node_product_name}",
+            "name": "%s %s" % (self.values.primary.node.node_manufacturer_name,
+                               self.values.primary.node.node_product_name),
             "manufacturer": self.values.primary.node.node_manufacturer_name,
             "model": self.values.primary.node.node_product_name,
         }
 
     @property
     def device_state_attributes(self):
         """Return the device specific state attributes."""
-        attrs = {const.ATTR_NODE_ID: self.values.primary.node.node_id}
+        attrs = {
+            const.ATTR_NODE_ID: self.values.primary.node.node_id,
+        }
+        if self.power_consumption is not None:
+            attrs[ATTR_POWER] = self.power_consumption
 
         return attrs
 
     @property
     def name(self):
         """Return the name of the entity."""
-        return f"{self.values.primary.node.node_manufacturer_name} {self.values.primary.node.node_product_name}: {self.values.primary.label}"
+        return "%s %s: %s" % (self.values.primary.node.node_manufacturer_name,
```

Comment:
Use modern string formatting: `f"{node.node_manufacturer_name} {node.node_product_name} {primary_value.label}"`

Reply:
OK, adjusted it to shorten the line. Will put back the modern string formatting with a line break ?